src/components/Class/index.js

```diff
@@ -178,6 +192,31 @@ function Class({ classToEdit }) {
     }
   };
 
+  const changeHandler = async (e, setField, field) => {
```

Comment:
this handler is not needed, just use `setFormField`. If you want to set two fields, you can do:

setFormField(e.target.value, [TYPE]);
setFormField('', 'until')

Reply:
This we need to use to handle occurrence and until. If a user is typing in the field until and the in occurrence then until field should get empty and if the user first types in occurrence and the until then occurrence field should get empty. The rest of the things will resolve and will let you know.